docs/2.10.0/docs/canton/usermanual/packagemanagement.rst

```diff
@@ -61,235 +61,183 @@ Understanding Package Vetting
 -----------------------------
 
 Every participant operator uploads DARs individually to their participant node. There is no global DAR repository
-anywhere and participants do not have access to each others DAR repositories. Therefore, for two participants to
-synchronise on a transaction that uses packages contained in a certain DAR, we need both participant operators to
-have uploaded the same DAR before the transaction was submitted.
+anywhere and participants do not have access to each others DAR repositories. Instead, participants publicly declare towards their peers
+that they are ready to receive transactions referring to certain packages.
+This is done by Daml package state topology transactions which can be seen by all the participants connected to a specific domain.
+Therefore, for two participants to synchronise on a transaction that uses packages contained in a certain DAR, both participant operators
+must upload and enable vetting for the same DAR and its contained packages before the transaction is submitted.
 
-If one of the involved participants doesn't know about a certain DAR, then the transaction will bounce with an error
+If one of the involved participants doesn't know about the packages of a certain DAR, then the transaction will bounce with an error
 ``NO_DOMAIN_FOR_SUBMISSION`` (with additional metadata listing which package has not been vetted on which participant).
-
-This error goes back to the fact that both participants not only upload the DAR, but also publicly declare towards
-their peers that they are ready to receive transactions referring to certain packages. This declaration happens
-automatically when you upload a DAR. The package vettings can be inspected using (preview):
-
-.. snippet:: package_dar_removal
-    .. success(output=8):: participant2.topology.vetted_packages.list()
-
-Vetting is necessary, as otherwise, a malicious participant might send a transaction referring to package a receiver
-does not have, which would make it impossible for the receiver to process the transaction, leading to a ledger fork.
-As transactions are valid only if all involved participants have vetted the used packages, this attack cannot happen.
-
-Removing Packages and DARs
---------------------------
+This can happen if the DAR has not been uploaded to the participant node or it has not been vetted.
```

Comment:
This sentence is repeating what the previous one has already said. Unless there is a distinction between "participants doesn't know about the package" and "DAR has not been uploaded to the participant node or it has not been vetted". If we are just clarifying what "doesn't know" means, I would just inline that.